_posts/2016-12-19-ml-india-nips-conference-2016-highlights.md

```diff
@@ -22,6 +22,9 @@ We present here some notes from the conference prepared by various researchers -
 
 - Google managed to get in a number of publications this year. Click [here](https://research.googleblog.com/2016/12/nips-2016-research-at-google.html){:target="_blank"} to read their blog describing some of their work.
 
+- This blog presents a conference report by Andreas Stuhlmülle:   
```

Comment:
@nishankvboy add a line acknowledging dr. biplav here.
something like 

[Reference credit: <href>Dr. Biplav Srivastava</href>] This blog presents a conference report by blah blah